server/src/feature_cache.rs

```diff
@@ -67,6 +67,23 @@ impl FeatureCache {
         self.send_full_update(key);
     }
 
+    pub fn apply_delta(&self, key: String, delta: &ClientFeaturesDelta) {
+        let client_features = ClientFeatures {
+            version : 1,
```

Comment:
```suggestion
            version : 2,
```

Honestly we've been on version 2 for so long I don't even know what version 1 does

Reply:
Should this eventually come from Unleash itself? If we ever change this version we'll get out of sync.